src/components/modal/v2/parts/Header.jsx

```diff
@@ -4,7 +4,17 @@ import { useRef, useState } from 'preact/hooks';
 import { useTransitionState } from '../lib';
 import Icon from './Icon';
 
-const Header = ({ headline, subheadline, className = '', logo, contentWrapper, contentBodyRef, contentBackground }) => {
+const Header = ({
+    headline,
+    subheadline,
+    className = '',
+    logo,
+    contentWrapper,
+    contentBodyRef,
+    contentBackground,
+    isQualifying = 'false',
```

Comment:
```suggestion
    isQualifying = false,
```
Let's change this to a string false to a boolean false since the check was actually failing.